### PR TITLE
[JEP-207] - Copy-edit design decisions and body

### PR DESCRIPTION
**Pull request by oleg-nenashev**
CC @jglick @bitwiseman

### CODE REVIEW

File: jep/207/README.adoc [hunk 1]
```diff
@@ -85,15 +85,17 @@ Support of other logging types will be documented in subsequent JEPs.
...
+* Freestyle and Jenkins Pipeline projects are supported out of the box
+* Run console logs should be provided using standard Jenkins interfaces
+* Data migration flow for upgrading instances is technically possible
+** It is OK if it requires special actions outside Jenkins
 * Reference External Build logging implementation provides a feasible performance and fault-tolerance compared to the original filesystem based solution
-* All Global Configurations are designed in a way that they can be configured via Jenkins Configuration-as-Code Plugin OOTB[e][f]
-Presumptions, obstacles and limitations (braindump)
+* All Global Configurations are designed in a way that they can be configured via Jenkins Configuration-as-Code Plugin
+
+=== Design decisions
```
Comment (jglick) [inline jep/207/README.adoc]: Better to follow the standard JEP structure: **Reasoning** and **Motivation**.

File: jep/207/README.adoc [hunk 2]
```diff
@@ -249,17 +273,21 @@ Get large text for the entire execution/run
...
-get lof as a Reader
-* `String getLog() throws IOException`
-* `List<String> getLog(int maxLines) throws IOException`
+get the log as a Reader
+* `String getLog() throws IOException` -
+gets the entire log as a single String
+** This method is deprecated in `hudson.model.Run`,
+and it should remain deprecated
+* `List<String> getLog(int maxLines) throws IOException` -
+gets a number of log lines as a
```
Comment (jglick) [inline jep/207/README.adoc]: as a…?

File: jep/207/README.adoc (hunk 1 above)
Comment (oleg-nenashev) [reply jep/207/README.adoc]: I really cannot read JEPs when decision making goes after the specification. I would prefer to keep it here unless it violates JEP requirements

Comment (bitwiseman) [reply jep/207/README.adoc]: @oleg-nenashev 
The choice of top level sections and their order was discussed and agreed on.  It is set and should not be changed.     If it matters to you that it be another way, propose the change i
including bringing all other JEPs into compliance with the new structure.  


It was an mistake on my part as an editor that I didn't catch that this JEP does not meet the requirements.  I'll make the updates to bring it back into compliance.

Comment (bitwiseman) [reply jep/207/README.adoc]: @oleg-nenashev  
Looking now, I see that the reasoning for the section ordering is not mentioned in the JEP-1, but I know it was discussed.  I will go dig up the discussion and make sure it is added to the Reasoning.

Comment (oleg-nenashev) [reply jep/207/README.adoc]: Anyway, it is quite unrelated to this PR, it does not really change it.
So I would prefer to merge it as is and then discuss.

**Comment by bitwiseman** [comment]
@oleg-nenashev 
I have merged this PR.  Please submit a new one to bring the JEP structure into compliance.

**Pull request merged**